avm/res/signal-r-service/signal-r/main.bicep

```diff
@@ -200,7 +225,7 @@ resource avmTelemetry 'Microsoft.Resources/deployments@2024-03-01' = if (enableT
   }
 }
 
-resource signalR 'Microsoft.SignalRService/signalR@2022-02-01' = {
+resource signalR 'Microsoft.SignalRService/signalR@2024-03-01' = {
```

Comment:
Any new features with this version other than managed identity? 
Not necessarily for this PR, just worth checking while upgrading the API version

Reply:
I reverted the version, as this is PR is only for the one desired new feature.